<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationApplyMain</class>
 <widget class="QWidget" name="CalibrationApplyMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1180</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1230</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>625</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>550</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>680</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_mid" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>360</width>
     <height>625</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>101</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>680</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_mid
{
border-left: 0px solid #B4B7BC;
background: #FFFFFF;
border-radius: 10px;
}

</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>280</width>
      <height>625</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>560</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>100000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget#tableWidget
{
font-family: PingFang SC;
font-weight: bold;
font-size: 26px;
line-height: 35px;
background: rgba(242,247,251,0.37);
border-radius: 10px;
border: 1px solid rgba(170,189,212,0.6);
}

QHeaderView {
    font-family: PingFang SC;
	font-weight: 500;
	font-size: 24px;
	color: #FFFFFF;
	line-height: 35px;
}

QHeaderView::section {
    background-color: #437AFF;
    border-top-left-radius: 10px;  
    border-top-right-radius: 10px;  
    padding: 5px;
    border: 1px solid #437AFF; 
    height: 35px; 
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_right" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>450</width>
     <height>650</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>396</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1400</width>
     <height>680</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_right
{
border-left: 0px solid #B4B7BC;
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>451</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget
{
border-bottom: 1px solid #B4B7BC;
background: #FFFFFF;

}</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>30</y>
       <width>240</width>
       <height>48</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 240px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>86</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>86</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 79px;
height: 20px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
     </property>
     <property name="text">
      <string>校准状态</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>100</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>392</width>
      <height>139</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2
{
background: #FFFFFF;
border-bottom: 1px solid #B4B7BC;
}</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>105</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 98px;
height: 20px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
     </property>
     <property name="text">
      <string>校准品批号</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_switch_cal" native="true">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>0</y>
       <width>321</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>86</y>
       <width>240</width>
       <height>48</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 241px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>68</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 58px;
height: 20px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
     </property>
     <property name="text">
      <string>有效期</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>431</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>392</width>
      <height>336</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLabel" name="label_c2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 26px;
height: 16px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>C2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edit_c5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 310px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);
width: 108px;
height: 18px;
font-family: PingFang SC;
font-weight: bold;
font-size: 23px;
color: #333333;
line-height: 43px;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edit_c2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 310px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);
width: 108px;
height: 18px;
font-family: PingFang SC;
font-weight: bold;
font-size: 23px;
color: #333333;
line-height: 43px;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edit_c3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 310px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);
width: 108px;
height: 18px;
font-family: PingFang SC;
font-weight: bold;
font-size: 23px;
color: #333333;
line-height: 43px;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_c3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 22px;
height: 16px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>C3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_c4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 22px;
height: 16px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>C4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_c0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 310px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);
width: 108px;
height: 18px;
font-family: PingFang SC;
font-weight: bold;
font-size: 23px;
color: #333333;
line-height: 43px;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edit_c4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 310px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);
width: 108px;
height: 18px;
font-family: PingFang SC;
font-weight: bold;
font-size: 23px;
color: #333333;
line-height: 43px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_c1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 22px;
height: 16px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>C1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_c1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>309</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 310px;
height: 48px;
background: #F2F7FB;
border-radius: 24px;
border: 1px solid rgba(170,189,212,0.6);
width: 108px;
height: 18px;
font-family: PingFang SC;
font-weight: bold;
font-size: 23px;
color: #333333;
line-height: 43px;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_c5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 22px;
height: 16px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>C5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_c0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 22px;
height: 16px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 35px;</string>
       </property>
       <property name="text">
        <string>C0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
